### DISCUSSION TITLE
How to operate a menu with the mouse?

### DISCUSSION THREAD

**Discussion by aldebaran-Wildt** [StereoKit/StereoKit#549] [Q&A] [answered]
Hello,
I am totally new to StereoKit (like it very much so far!), an just created my first window with a toggle and a slider. When I start the app on my Windows computer I can move the window with the mouse (drag&drop), but I haven't found out how to toogle the button or move the slider with the mouse (tried various combinations with shift/ctrl/alt and mouse left/right/wheel etc.). 
What do I miss?
Thanks!
Usch

**Comment by maluoi** [comment, marked as answer]
Sounds like you've found most of the controls! There's a [document here](https://stereokit.net/Pages/Guides/Using-The-Simulator.html) on how to use the simulator that may provide further insight.

Mostly though, it's about positioning the hand properly. You do need to push the hand's fingers _through_ the button using the scroll wheel, and similar for the slider!

**Reply by aldebaran-Wildt** [reply]
Thank you!
